<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteSettingsDialog</class>
 <widget class="QWidget" name="RemoteSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>51</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
     <item>
      <widget class="RemoteSettingsCommandWidget" name="remoteSettingsCommandWidget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="remoteSyncButton">
       <property name="text">
        <string>Sync</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remoteSaveButton">
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RemoteSettingsCommandWidget</class>
   <extends>QWidget</extends>
   <header>gui/remote/RemoteSettingsCommandWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
